src/style.css

```diff
@@ -0,0 +1,425 @@
+    background: var(--bg-modal);
+  }
+}
+
+@keyframes fadOutnProject {
+  0% {
+    z-index: 20;
+    background: var(--bg-modal);
+  }
+
+  100% {
+    z-index: -3;
+    background: var(--bckgrd-0);
+  }
+}
+
+.app-item-popup {
+  padding-inline: 2.5vw;
+  display: flex;
+  flex-flow: column;
+  justify-content: center;
+  background-color: var(--bckgrd);
+  border: 1px solid var(--border);
+  border-radius: 16px;
+  padding-block: 2.5vh;
+  gap: 10px;
+  max-height: 85vh;
+  transform: scale(0.3);
+  animation: scaleUpProject 0.3s cubic-bezier(0.165, 0.84, 0.44, 1) forwards;
+}
+
+.app-info {
+  display: flex;
+  flex-flow: column;
+  justify-content: center;
+  gap: 10px;
+}
+
+.out .app-item-popup {
+  animation: scaleDownProject 0.3s ease-out forwards;
+}
+
+@keyframes scaleUpProject {
+  0% {
+    transform: scale(0.3) translateY(1000px);
+    opacity: 0;
+  }
+
+  100% {
+    transform: scale(1) translateY(0);
+    opacity: 100%;
+  }
+}
+
+@keyframes scaleDownProject {
+  0% {
+    transform: scale(1) translateY(0);
+    opacity: 100%;
+  }
+
+  100% {
+    transform: scale(0.3) translateY(1000px);
+    opacity: 0;
+  }
+}
+
+.app-exit,
+.app-title-pop {
+  font-weight: bolder;
+}
+
+.app-exit {
+  text-align: right;
+  cursor: pointer;
+  font-size: 1.5rem;
+}
+
+.app-property {
+  --wap: 800px;
+
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(clamp(100%/3 + 0.1%, (var(--wap) - 100vw) * 1000, 100%), 1fr));
+  row-gap: 5px;
+  flex-shrink: 2;
+  font-size: 0.85rem;
+}
+
+.app-img-pop {
+  max-height: 35vh;
+  margin-inline: auto;
+  max-width: 28vw;
+}
+
+.app-title-header {
+  display: grid;
+  grid-template-columns: 1fr auto;
+  align-items: center;
+  font-size: 0.75rem;
+}
+
+/* Comments and Reservation Section */
+
+.app-sec {
+  display: flex;
+  flex-flow: column;
+  justify-content: space-around;
+  gap: 10px;
+}
+
+.app-section-header {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.app-section-ctn {
+  max-height: 10vh;
+  overflow-y: scroll;
+  font-size: 0.85rem;
+}
+
+/* Comments and Reservation Form */
+
+.app-form-title {
+  text-align: center;
+}
+
+.app-form-ctn {
+  display: flex;
+  flex-flow: column;
+  gap: 7px;
+}
+
+.app-form-input-div {
+  display: flex;
+  flex-flow: column;
+  gap: 7px;
+}
+
+.app-form-input-reg,
+.app-form-textArea {
+  padding-inline: 7px;
+  padding-block: 5px;
+  z-index: 1;
+  border-radius: 10px;
+}
+
+.app-form-textArea {
+  resize: none;
+}
+
+@media only screen and (min-width: 600px) {
+  .nav-bar {
+    flex-flow: row;
+    justify-content: space-between;
+  }
+
+  .app-form-input-div {
+    display: grid;
+    grid-template-columns: 1fr 3fr;
+    padding-inline: 0 5%;
+  }
+
+  .app-div-btn {
+    display: flex;
+    flex-flow: row;
+    flex-wrap: wrap;
+    gap: 2.5%;
+    justify-content: center;
+  }
+}
+
+@media (orientation: landscape) {
+  .app-item-popup-overlay {
+    padding-inline: 7.5vw 7.5vw;
+    padding-block: 20vh;
+  }
```

Comment:
Great design team🤝

- [ ] Great job on trying to fix the changes requested by the previous reviewer. However, your design has to be a replica of the design given. Kindly ensure your design resembles the design given.
- Your design
![Your design](https://github.com/lordksix/Meals-API-template/assets/63917725/06b57961-5e2b-41b4-9377-abafad9dd82f)
- Given design
![Given design](https://github.com/lordksix/Meals-API-template/assets/63917725/66aeb70c-b775-4067-ac93-1abe7b1233cb)

Reply:
Thank you for the feedback. I have eliminated the landscape mode of the layout as requested in commits c865458540a97dd0eb6545ae3f14f944a0a17e49 and 38bcbc9dfb729694fbbd27688b6a95f9423f3553